### PR TITLE
[zh] translate task certificates

### LINKED ISSUE #26807: [zh] translate "tasks/Certificates"
**This is a Feature Request**
translate docs/tasks/administer-cluster/certificates.md into Chinese

**What would you like to be added**
content/zh/docs/tasks/administer-cluster/certificates.md

/assign
/language zh

### PR DESCRIPTION
**Pull request by zhiguo-lu**
translate /docs/tasks/administer-cluster/certificates.md into Chinese.

close #26807

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/certificates.md [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+---
+-->
+
+<!-- overview -->
+
+<!-- 
+When using client certificate authentication, you can generate certificates
+manually through `easyrsa`, `openssl` or `cfssl`.
+-->
+在使用客户端证书认证的场景下，你可以通过 `easyrsa`、 `openssl` 或 `cfssl` 等工具，以手工方式生成证书。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/certificates.md:20]:
```suggestion
在使用客户端证书认证的场景下，你可以通过 `easyrsa`、`openssl` 或 `cfssl` 等工具以手工方式生成证书。
```

File: content/zh/docs/tasks/administer-cluster/certificates.md [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+        "DNS:kubernetes.default.svc.cluster,"\
+        "DNS:kubernetes.default.svc.cluster.local" \
+        --days=10000 \
+        build-server-full server nopass
+
+    <!-- 
+    1.  Copy `pki/ca.crt`, `pki/issued/server.crt`, and `pki/private/server.key` to your directory.
+    1.  Fill in and add the following parameters into the API server start parameters:
+    -->
+1.  拷贝 文件 `pki/ca.crt`、 `pki/issued/server.crt`、和 `pki/private/server.key` 到你的目录。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/certificates.md:80]:
```suggestion
1.  拷贝 文件 `pki/ca.crt`、`pki/issued/server.crt` 和 `pki/private/server.key` 到你的目录中。
```

File: content/zh/docs/tasks/administer-cluster/certificates.md [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+    1.  Create a config file for generating a Certificate Signing Request (CSR).
+    Be sure to substitute the values marked with angle brackets (e.g. `<MASTER_IP>`)
+    with real values before saving this to a file (e.g. `csr.conf`).
+    Note that the value for `MASTER_CLUSTER_IP` is the service cluster IP for the
+    API server as described in previous subsection.
+    The sample below also assumes that you are using `cluster.local` as the default
+    DNS domain name.
+    -->
+1.  创建一个用于生成证书签名请求（CSR）的配置文件。
+    保存文件(例如：`csr.conf`)前，记得用真实值替换掉尖括号中的值（例如：`<MASTER_IP>`）。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/certificates.md:125]:
```suggestion
    保存文件（例如：`csr.conf`）前，记得用真实值替换掉尖括号中的值（例如：`<MASTER_IP>`）。
```

File: content/zh/docs/tasks/administer-cluster/certificates.md [hunk 4]
```diff
@@ -0,0 +1,366 @@
...
+        chmod +x cfssl
+        curl -L https://github.com/cloudflare/cfssl/releases/download/v1.5.0/cfssljson_1.5.0_linux_amd64 -o cfssljson
+        chmod +x cfssljson
+        curl -L https://github.com/cloudflare/cfssl/releases/download/v1.5.0/cfssl-certinfo_1.5.0_linux_amd64 -o cfssl-certinfo
+        chmod +x cfssl-certinfo
+
+    <!-- 
+    1.  Create a directory to hold the artifacts and initialize cfssl:
+    -->
+1.  创建一个目录，用它保存构件和初始化 cfssl：
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/certificates.md:216]:
```suggestion
1.  创建一个目录，用它保存所生成文件和初始化 cfssl：
```

Comment (zhiguo-lu) [reply content/zh/docs/tasks/administer-cluster/certificates.md]: 高质量的 review，thanks

File: content/zh/docs/tasks/administer-cluster/certificates.md [hunk 5]
```diff
@@ -0,0 +1,366 @@
...
+    <!-- 
+    1.  Generate the certificate signing request based on the config file:
+    -->
+1.  基于上面的配置文件生成证书签名请求：
+
+        openssl req -new -key server.key -out server.csr -config csr.conf
+
+    <!-- 
+    1.  Generate the server certificate using the ca.key, ca.crt and server.csr:
+    -->
```
Comment (tanjunchen) [inline content/zh/docs/tasks/administer-cluster/certificates.md:172]: 漏了 ca.crt

Comment (tanjunchen) [reply content/zh/docs/tasks/administer-cluster/certificates.md:172]: /cc @zhiguo-lu

Comment (zhiguo-lu) [reply content/zh/docs/tasks/administer-cluster/certificates.md:172]: 收到，已改，谢谢

**Review by tanjunchen — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**